src/staking-v3/hooks/useDappStaking.ts

```diff
@@ -22,7 +22,13 @@ import {
 import { Symbols } from 'src/v2/symbols';
 import { ExtrinsicStatusMessage, IEventAggregator } from 'src/v2/messaging';
 import { useStore } from 'src/store';
-import { useAccount, useChainMetadata, useNetworkInfo, useLedger } from 'src/hooks';
+import {
+  useAccount,
+  useChainMetadata,
+  useNetworkInfo,
+  useLedger,
+  ETHEREUM_EXTENSION,
```

Comment:
I think exporting constant in hooks is not a good way, I think moving `ETHEREUM_EXTENSION` to another files like `src/constants/~~` but I want to know what you thinking about it

Reply:
You are right. Please refer to this commit.